File: Python Exams/Basic Exam Collection/Basics Online Exam - 15 and 16 June 2019/05. Oscars/05. Oscars.py
artist_name = input()
artist_points = float(input())
jury_list = int(input())

needed_point = 1250.5

for _ in range(jury_list):
    jury_name = input()
    jury_point = float(input())

    artist_points += len(jury_name) * jury_point / 2

    if artist_points > needed_point:
        print(f'Congratulations, {artist_name} got a nominee for leading role with {artist_points:.1f}!')
        exit()

print(f'Sorry, {artist_name} you need {needed_point - artist_points:.1f} more!')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 15 and 16 June 2019/05. Oscars/05. Oscars.js
function oscars(arg) {
    let artistName = arg.shift()
    let artistPoints = +arg.shift()
    let juryCount = +arg.shift()
    let neededPoint = 1250.5

    for (let i = 0; i < juryCount; i++) {
        let juryName = arg.shift()
        let juryPoint = +arg.shift()
        artistPoints += juryName.length * juryPoint / 2
        if (artistPoints >= neededPoint)
            break
    }

    if (artistPoints > neededPoint)
        return `Congratulations, ${artistName} got a nominee for leading role with ${artistPoints.toFixed(1)}!`
    return `Sorry, ${artistName} you need ${(neededPoint - artistPoints).toFixed(1)} more!`
}

// console.log(oscars([
//     'Sandra Bullock',
//     '340',
//     '5',
//     'Robert De Niro',
//     '50',
//     'Julia Roberts',
//     '40.5',
//     'Daniel Day-Lewis',
//     '39.4',
//     'Nicolas Cage',
//     '29.9',
//     'Stoyanka Mutafova',
//     '33',

// ]))